site/docs/master/release-instructions.md

```diff
@@ -37,9 +38,8 @@ To run the `goreleaser` process to generate a GitHub release, you'll need to hav
 You may regenerate the token for every release if you prefer.
 
 #### If you don't already have a token
-1.  Go to https://github.com/settings/tokens/new.
+1.  Go to https://github.com/settings/tokens/new?scopes=repo.
```

Comment:
Added this to the link and removed the 3rd step to make things a little more straightforward after I saw how Homebrew managed it.